react/tic-tac-toe/src/app/screens/Game/components/MatchHistory/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+import services from '~services/MatchesService'; // eslint-disable-line import/no-unresolved
+
+import styles from './styles.module.scss';
+
+class MatchHistory extends Component {
+  state = {
+    matches: []
+  }
+
+  componentDidMount() {
+    services.getMatches().then(response => this.setState({ matches: response.data }));
+  }
+
+
+  humanize = (property) => property.replace(/_/g, ' ')
```

Comment:
This is more workload than you need, just use a dictionary por matching the correct player or Tie